### PR TITLE
Remove npm tarball and rework npm organization

### PR DESCRIPTION
**Pull request by amyrlam**
## :pushpin: Summary

We want `@hashicorp/ember-flight-icons` to consume `@hashicorp/flight-icons`. But for now, this is a step in the right direction that makes the npm process slightly less confusing. Happy to walk through or pair on the process with anyone also!

Review commit-by-commit, but the second commit is so large, once you get to the .svg's you kinda have the gist of it.

### :speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

Examples: 
- issue (ux,non-blocking): These buttons should be red, but let's handle this in a follow-up.
- suggestion (non-blocking): Let's change this wording to make it easier to understand.
- issue (blocking): We shouldn't introduce this kind of tech debt; let's pair and resolve the issue in a more sustainable way.

## :link: External Links

Resolves: https://github.com/hashicorp/flight/issues/108

## 👉 Next steps

- [ ] TODO: Publish a new version and test to make sure things are working well
- [ ] TODO: File any follow-ups to this task 
  - I think ideally we want @hashicorp/ember-flight-icons to consume @hashicorp/flight-icons
  - Stuff could probably be more automated with GitHub actions

### CODE REVIEW

**Comment by amyrlam** [comment]
cc @randallmorey I know you had filed https://github.com/hashicorp/flight/issues/90, if you have any opinions on the file organization let me know! All the files will now live in `/flight-icons/icons/` so will be a slight tweak in usage on the next npm release. 

I haven't published a new npm version yet. Will wait for feedback next week on this PR, which I only consider a (slight) step in the right direction.

**Review by Dhaulagiri — COMMENTED** [review]
Added one suggestion and one question, the answer to which could elicit more feedback

File: README.md [hunk 1]
```diff
@@ -26,35 +26,40 @@ This addon can be used in React or Ember apps.
...
-cd ember-flight-icons/public/icons
+cd flight-icons
 ```
 
 ```bash
 npx spritesh
 ```
 
-## How to make the `@hashicorp/ember-flight-icons` npm package
+The `spitesh` command will great a `sprite.svg`. Copy this file to ember-flight-icons/
```
[the remark this answers is not in the document]
Comment (randallmorey) [reply README.md:51]: What does `spritesh` mean?

[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply README.md:51]: https://www.npmjs.com/package/spritesh

File: flight-icons/package.json [hunk 2]
```diff
@@ -1,37 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (amyrlam) [reply flight-icons/package.json]: yes, that was a mistake (thanks for catching that!) and i squashed it in

**Comment by amyrlam** [comment]
@Dhaulagiri updated... ideally, if we merge this, i can bump both package.json's separately (or as part of this PR), npm publish, and confirm that the addons still work. seems ok to bump liberally since we're still in beta semver?

File: README.md [hunk 3]
```diff
@@ -26,35 +26,40 @@ This addon can be used in React or Ember apps.
 
 1. For export-go, the Figma export tool, see [export-go/README](export-go/README.md).
 
-1. On every new Figma export, we need to manually copy the icons to the `ember-flight-icons/public/icons/` directory right now.
+1. On every new Figma export, we need to manually copy the icons to the `flight-icons/` directory right now.
```
Comment (Dhaulagiri) [inline README.md:29]:
```suggestion
1. On every new Figma export, we need to manually copy the icons to the `flight-icons/icons` directory right now.
```

File: README.md [hunk 4]
```diff
@@ -26,35 +26,40 @@ This addon can be used in React or Ember apps.
...
 
 ```bash
-rm ember-flight-icons/public/icons/*
+rm flight-icons/*
```
Comment (Dhaulagiri) [inline README.md:32]:
```suggestion
rm flight-icons/icons/*
```

File: README.md [hunk 5]
```diff
@@ -26,35 +26,40 @@ This addon can be used in React or Ember apps.
...
 ```
 
 ```bash
-cp export-go/dist/* ember-flight-icons/public/icons/
+cp export-go/dist/* flight-icons/
```
Comment (Dhaulagiri) [inline README.md:36]:
```suggestion
cp export-go/dist/* flight-icons/icons
```

File: README.md [hunk 6]
```diff
@@ -26,35 +26,40 @@ This addon can be used in React or Ember apps.
...
 ```
 
 ### sprite instructions
 
 1. To generate the sprite file, [sprite.svg](ember-flight-icons/public/icons/sprite.svg), follow the instructions in the README of [edenspiekermann/sprite.sh](https://github.com/edenspiekermann/sprite.sh) e.g.
 
 ```bash
-cd ember-flight-icons/public/icons
+cd flight-icons
```
Comment (Dhaulagiri) [inline README.md:44]:
```suggestion
cd flight-icons/icons
```

File: README.md [hunk 7]
```diff
@@ -26,35 +26,40 @@ This addon can be used in React or Ember apps.
...
 ```bash
 npx spritesh
 ```
 
-## How to make the `@hashicorp/ember-flight-icons` npm package
+The `spitesh` command will great a `sprite.svg`. Copy this file to ember-flight-icons/
 
 ```bash
-cd ember-flight-icons && npm pack
+cd .. && cp flight-icons/sprite.svg ember-flight-icons/public/ && cp flight-icons/_catalog.json ember-flight-icons/public/
```
Comment (Dhaulagiri) [inline README.md:54]:
```suggestion
cd .. && cp flight-icons/icons/sprite.svg ember-flight-icons/public/ && cp flight-icons/icons/_catalog.json ember-flight-icons/public/
```

**Review by Dhaulagiri — COMMENTED** [review]
If I'm understanding where we at with the current layout I think we want to make these changes and also update the package.json to have 
```
"files": [
    "icons"
  ],
```

**Review by Dhaulagiri — APPROVED** [review]

**Comment by amyrlam** [comment]
talked to brian in dm. to de-complicate things, i'll merge this and later today, publish a new version of both and confirm the instructions work

**Pull request merged**